conf/documentation.conf

```diff
@@ -303,28 +303,35 @@ EOT
 type=toggle
 options=enabled|disabled
 description=<<EOT
-Should winbindd be started ? Keep enabled unless you know what you're doing.
+Should winbindd be started? Keep enabled unless you know what you're doing.
 EOT
 
 [services.redis_cache]
 type=toggle
 options=enabled|disabled
 description=<<EOT
-Should redis for caching be started ? Keep enabled unless you know what you're doing.
+Should redis for caching be started? Keep enabled unless you know what you're doing.
 EOT
 
 [services.redis_queue]
 type=toggle
 options=enabled|disabled
 description=<<EOT
-Should redis be started ? Keep enabled unless you know what you're doing.
+Should redis be started? Keep enabled unless you know what you're doing.
+EOT
+
+[services.redis_ntlm_cache]
```

Comment:
At the most obvious place :P in the untracked files of my servers.

Pushing it right now